### DISCUSSION TITLE
Mame 2003 arcade games not working in Emu-js need help!!!

### DISCUSSION THREAD

**Discussion by Wren-west** [EmulatorJS/EmulatorJS#534] [Q&A]
can you tell me the problem? I have downloaded many mame roms and tested them on my website and they are not loading!!!
The error msg: required files missing game can not be loaded!

**my Emulatorjs settings are:**
`<script type='text/javascript'>
    EJS_player = '#game';
    
    // Can also be fceumm or nestopia
    EJS_core = 'mame2003';
    
	
	EJS_color = "grey";
    
	EJS_startOnLoaded = true;
	
	 EJS_gameParentUrl = 'https://<mywebsite>.com/games/rom/neogeo/mslug2.zip'; // Url to Parent Rom file
    
    // URL to BIOS file
    EJS_biosUrl = 'https://<mywebsite>.com/games/rom/neogeo/neogeo.zip'; // example: https://dl.dropboxusercontent.com/s/[random-code]/bios.bin
    
    // URL to Game rom
    EJS_gameUrl = 'https://<mywebsite>/games/rom/neogeo/mslug2.zip'; // example: https://dl.dropboxusercontent.com/s/[random-code]/mario.nes
    
    // Path to the data directory
    EJS_pathtodata = 'https://<mywebsite>.com/games/data/';
</script>`

Here is the error screenshot:
[https://ibb.co/JFMzF4d](https://ibb.co/JFMzF4d)

**Comment by ethanaobrien** [comment]
>Here is the error screenshot:
https://ibb.co/JFMzF4d

Please upload the image straight to github. I don't go to external sites to view images. There is a "click to upload file" bar right under the text box in github

**Comment by Wren-west** [comment]
![Uploading error.png…]()
